### DISCUSSION TITLE
Can I in some way install just Koreader on a Kobo (model N613) and erase everything else ?

### DISCUSSION THREAD

**Discussion by sherifhendrik** [koreader/koreader#12006] [General]
I love koreader, it has become my only way of reading files in a screen and organizing an extensive collection of epubs, pdfs, .txt files and rss feeds. I love Koreader so much that i would like to have ONLY it show up on my Kobo when turning on and off. I don't care about other extensions, and i proactivelly want to erase all rakuten code stuff from my device for privacy concerns. Not to mention that turning directly into Koreader would be much faster than dealing with the home screen. I only side-load to it, or download from wikipedia or rss through koreader. So, is there any way to have only Koreader stuff on my Kobo ?

**Comment by Frenzie** [comment]
For some devices there is https://github.com/lgeek/okreader but generally speaking I think we can safely say there's just no need to ever turn it off (and deal with startup) as long as you use it once every few days.

**Reply by sherifhendrik** [reply]
thanks, i will try this out. if i am not successful, i will do the second suggestion.

**Reply by jonnyl2** [reply]
Just wondering (@Frenzie or anyone) – what would you do if an update or some hack causes KOReader to crash on start-up? Would you still be able to connect it to a computer somehow? Or would you have to do a hard reset of the entire device?

**Reply by Frenzie** [reply]
I don't see any point in getting rid of Nickel.[1] Just disable auto-update so things don't randomly happen if you do happen to be in it. Sometimes months or perhaps even years go by that I never even see it; it might as well not be there. Once in a blue moon I might buy and read a book in the Kobo store to see how things progressed from last time.

> Or would you have to do a hard reset of the entire device?

A hard reset would mean replacing the SD card, yes. But you might make a kind of failsafe part of the startup script, to always connect to wifi so you can connect over SSH for example.

[1] From a KOReader perspective. Of course there are many reasons like curiosity etc.

**Reply by jonnyl2** [reply]
I agree. The only time I use Nickel is when some user patch or other hack causes KOReader to crash on launch. Then I can easily fix the problem via Nickel's USB connection.
